mozjs/src/generate_wrappers.sh

```diff
@@ -24,14 +24,16 @@ grep_heur() {
   $gsed 's/mozilla:://g' |
   $gsed 's/Handle<\*mut JSObject>/HandleObject/g' |
   grep -F -v '> HandleObject' | # We are only wrapping handles in args not in results
+  grep -F -v '> HandleValue' |
+  grep -F -v '> HandleString' |
   grep -v 'MutableHandleObjectVector' # GetDebuggeeGlobals has it
 }
 
 # usage find_latest_version_of_file_and_parse $input_file $out_wrapper_module_name
 find_latest_version_of_file_and_parse() {
   # clone file and reformat (this is needed for grep_heur to work properly)
   # this $(find) only gets last modified file
-  cp $(find target -name "$1" -printf "%T@ %p\n" | sort -n | tail -n 1 | tr ' ' '\n' | tail -n 1) "target/wrap_$1"
```

Comment:
we should probably make this var like we do for `gsed`

Reply:
Done.